dp-hello-world-event/event/handler.go

```diff
@@ -0,0 +1,27 @@
+package event
+
+import (
+	"context"
+	"fmt"
+	"github.com/ONSdigital/log.go/log"
+)
+
+// HelloCalledHandler ...
+type HelloCalledHandler struct {
+}
+
+// Handle takes a single event.
+func (h *HelloCalledHandler) Handle(ctx context.Context, event *HelloCalled) (err error) {
+	logData := log.Data{
+		"event": event,
+	}
+	log.Event(ctx, "event handler called", log.INFO, logData)
+
+	//TODO Replace with actual event handler logic…
```

Comment:
I guess this `TODO` will be done in a different PR?

Reply:
It was meant to be a TODO for anyone expanding on the example, generated code. (Ie. when someone has done project generation, this is where the handler logic needs to go.)